src/main/java/PluginBukkitBridge/FakeConsoleCommandSender.java

```diff
@@ -136,4 +138,8 @@ public void sendRawMessage(String arg0) {
         MyPlugin.fixme();
     }
 
+    @Override
+    public Spigot spigot() {
+        return new Spigot();
```

Comment:
I suggest you redefine the the methods inherited from CommandSender.Spigot and either implement the functionality or call `MyPlugin.fixme()`. Otherwise calls to them would throw an `UnsupportedOperationException`.

Reply:
fixed